config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] When setting up routes in Rails, it's a best practice to be explicit about which actions you want to allow for each resource. Instead of using the blanket `resources :users do` followed by `resources :posts`, consider using a more targeted approach.
For instance, you can use `resources :users, only: [:index, :show] do` followed by `resources :posts, only: [:index, :show]` inside it. This way, you're clearly specifying that you only intend to enable the `index` and `show` actions for both users and their associated posts. This makes your codebase more organized and secure by limiting the exposed endpoints to only the necessary ones. Keep up the great coding! 🚀👨‍💻